x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/_classification_exploration.scss

```diff
@@ -32,7 +33,7 @@
 .mlDataFrameAnalyticsClassification__dataGridMinWidth {
   float: left;
   min-width: 480px;
-  width: 92%;
+  width: calc(100% - 80px);
```

Comment:
Minor suggestion you could set the 80px in a variable and then use it in these two places. Might save some hassle in the future should somebody want to change it. There may also be more instances of this value over time.